tests/test_arcaflow_plugin_uperf.py

```diff
@@ -82,11 +83,12 @@ def test_profile_gen(self):
         with open(uperf_plugin.profile_path, "r", encoding="us-ascii") as file:
             generated_file = file.read()
         uperf_plugin.clean_profile()
+        self.maxDiff = None
```

Comment:
Is `maxDiff` used?  If not, we should presumably revert this addition.

Reply:
That very much should stay. Every change that modifies the xml needs maxDiff set to make it viewable. It's test code, so there is no real overhead keeping it here.